includes/admin/payments/view-order-details.php

```diff
@@ -31,6 +31,12 @@
 	wp_die( __( 'The specified ID does not belong to an order. Please try again', 'easy-digital-downloads' ), __( 'Error', 'easy-digital-downloads' ) );
 }
 
+if ( 'refund' === $order->type ) {
+	require_once EDD_PLUGIN_DIR . 'includes/admin/payments/view-refund.php';
+	edd_view_refund_page_content();
+	return;
+}
```

Comment:
@robincornett I wonder if we should keep loading of the views to just the `edd_payment_history_page()` function? 

When I suggested checking the type I meant checking in the `wp_die()` condition above. I think since you can only get to the wrong page by manually changing the URL it's safe to put developer experience over user experience and present users with an "ugly" error message and keep our view loading logic contained.

cc @ashleyfae

Reply:
We are doing a similar thing for Refunds here: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/payments/view-refund.php#L29-L33

Reply:
I'd rather either do a redirect or a `wp_die()`.